cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -900,15 +878,9 @@ SwiftGenerator::writeMemberwiseInitializer(
 }
 
 void
-SwiftGenerator::writeMembers(
```

Comment:
This function has a 4th parameter: `int typeCtx` which had a default value of `0`.
None of the callers ever passed it, so `typeCtx` was always `0`.